src/main/java/no/ndla/taxonomy/service/dtos/SearchableTaxonomyContextDTOFactory.java

```diff
@@ -0,0 +1,83 @@
+package no.ndla.taxonomy.service.dtos;
+
+import no.ndla.taxonomy.domain.*;
+import no.ndla.taxonomy.repositories.RelevanceRepository;
+import no.ndla.taxonomy.rest.v1.dtos.nodes.searchapi.*;
+import org.springframework.stereotype.Service;
+
+import java.net.URI;
+import java.util.*;
+
+@Service
+public class SearchableTaxonomyContextDTOFactory {
```

Comment:
Noe usikker på om dette er rette måten å gjøre det på.
Trenger en måte å få tilgang på `relevanceRepository` når jeg bygger en `SearchableTaxonomyContextDTO` så jeg lagde en service-klasse for å gjøre det. Tenkte det var litt uheldig å koble på et repository på hver eneste `DTO` som blir lagd da den potensielt kan bli lagd en del.

Reply:
Dette er nok lurt. Potensielt stort minnebruk om du skal la spring autowire inn repositoriet i alle dtos.